esp/esp/program/modules/module_ext.py

```diff
@@ -137,7 +124,8 @@ def module(self):
         return self.program.getModule('StudentClassRegModule')
 
     def reg_verbs(self):
-        verb_list = [self.signup_verb]
+        verb_list = [RegistrationType.get_map(include=['Enrolled'],
```

Comment:
It looks like if you want you can just inline this to `RegistrationType.get_or_create(...)`.